docs/src/api.md

```diff
@@ -291,7 +291,7 @@ return df_agent, df_model
 
 ```@docs
 Schedulers
-schedule
+schedule(::ABM)
```

Comment:
┌ Warning: undefined binding 'schedule' in `@docs` block in src/api.md:292-295

Reply:
Is the method declaration with `::ABM`? Because scheduling only applies to the `StandardABM`.

Reply:
It's
``` julia

"""
    schedule(model::StandardABM) → ids

Return an iterator over the scheduled IDs using the model's default scheduler.
"""
schedule(model::ABM) = abmscheduler(model)(model)
```

Reply:
Oh it is because it is `Schedulers.schedule`. It is in a submodule.

Reply:
There is a comment just below this lines that says the opposite

Reply:
oh. Well, then I don't know why this is erroring :D